browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -256,6 +263,39 @@ void BraveTabContainer::UpdateLayoutOrientation() {
   InvalidateLayout();
 }
 
+void BraveTabContainer::PaintBoundingBoxForTiles(gfx::Canvas& canvas) {
+  auto* split_view_data =
```

Comment:
why is this not guarded by the flag?

Reply:
Rather than adding the flag everywhere, it seems better to just check for `nullptr` everywhere that `SplitViewBrowserData::FromBrowser` is used and definitely seems problematic to do neither

Reply:
Will follow up soon.